lib/main.js

```diff
@@ -96,6 +97,7 @@ globals.setImmediate(() => {
 });
 
 module.exports = runner.chain;
+module.exports.CallSite = CallSite;
```

Comment:
Fair enough. I'm in the middle of moving though, so it might be a week or so till I have the time to look at this again. By then maybe there's more feedback :)